Update README.md

tlimato opened the pull request:
Added Answer. Based this on my other proofs regarding isomorphism, and graphs.

tlimato committed:
Update README.md

added answer

larskotthoff commented on README.md:
```diff
@@ -9,3 +9,28 @@ might help with the notation for mathematical expressions.
+We can then state that  $|E_A|  =  |E_B|  =  \frac{n(n-1)}{2}$.
+
+3. **Showing  a Bijection**:
+Since  $|V_A|  =  |V_B|  = n$, there exists a bijection  $f: V_A  \rightarrow V_B$ that is possible because both sets of vertices have the same finite number of elements. By definition of isomorphism, this bijection  $f$ must satisfy the condition that  $(u,v)  \in E_A$ if and only if  $(f(u),f(v))  \in E_B$, applying for all arbitrary  $u$ and  $v$, with their corresponding  $f(u)$ and  $f(v)$,  ...
```
So how do you know that such a bijection exists?

tlimato committed:
Update README.md

Similar to the Other Isomorphism Problem I just read back up on my Notes, re-referenced the definition of Complete Graphs, and rewrote the entire proof. I apologize it took this long for me to get it right but it's too late for that now.

tlimato committed:
Update README.md

format fix

tlimato commented:
Similar to the Other Isomorphism Problem I just read back up on my Notes, re-referenced the definition of Complete Graphs, and rewrote the entire proof. I apologize it took this long for me to get it right but it's too late for that now.

larskotthoff reviewed (changes requested):
How do you get this bijection? You still haven't shown that it must exist.

tlimato committed:
Update README.md

added a very explicit explanation of why the properties of A and B mean any bijection that maps an edge in A to an edge in B is valid and must exist.

tlimato committed:
Update README.md

tlimato commented:
Added a very explicit explanation of why the properties of A and B mean any bijection that maps an edge in A to an edge in B is valid and must exist.

tlimato commented:
Furthermore, I use Github copilot to assist in writing code, It's built into my vscode install I use for Proffesional work and It's helpful in accelerating projects and avoiding syntax issues. I will make this clarification on other assignments as well.
